agreement.go

```diff
@@ -0,0 +1,103 @@
+package xinvoice
+
+import "github.com/invopop/gobl/bill"
+
+// Agreement defines the structure of the ApplicableHeaderTradeAgreement of the CII standard
+type Agreement struct {
+	BuyerReference string  `xml:"ram:BuyerReference,omitempty"`
+	Seller         *Seller `xml:"ram:SellerTradeParty"`
+	Buyer          *Buyer  `xml:"ram:BuyerTradeParty"`
+}
+
+// Seller defines the structure of the SellerTradeParty of the CII standard
+type Seller struct {
+	Name                      string                     `xml:"ram:Name"`
+	LegalOrganization         *LegalOrganization         `xml:"ram:SpecifiedLegalOrganization,omitempty"`
+	Contact                   *Contact                   `xml:"ram:DefinedTradeContact"`
+	PostalTradeAddress        *PostalTradeAddress        `xml:"ram:PostalTradeAddress"`
+	URIUniversalCommunication *URIUniversalCommunication `xml:"ram:URIUniversalCommunication>ram:URIID"`
+	SpecifiedTaxRegistration  *SpecifiedTaxRegistration  `xml:"ram:SpecifiedTaxRegistration>ram:ID"`
+}
+
+// Contact defines the structure of the DefinedTradeContact of the CII standard
+type Contact struct {
+	Name  string `xml:"ram:PersonName"`
```

Comment:
I'd maintain the same name here, i.e. `PersonName` for clarity, as "name" on its own can also imply "company name".